common/importer/chrome_importer_utils.cc

```diff
@@ -103,6 +116,18 @@ base::Value::List GetChromeSourceProfiles(
           base::Value::Dict entry;
           entry.Set("id", value.first);
           entry.Set("name", *name);
+          if (last_active_profiles)
+            entry.Set("last_active",
```

Comment:
For Chromium browsers, there should be the last used one as `last_used`. In my Chrome install, I have 2 profiles and the `last_active` is just an empty array? `[]` (weird behavior)

Reply:
ah- seems if you close using `Exit`, it does save `last_active_profiles` properly. If you close the window using the big X in the top right (on Windows), it saves `[]` for some reason.

Either way - just want to check: is this the behavior we want? (iterating an array). Or do we just want the `last_used` value? (ex: `Default`)

Reply:
One last comment (sorry) - I checked another profile which does NOT have multiple profiles. In that one, it seems that `last_used` does not exist. This must only exist in cases where you have more than one profile created

Reply:
Talked via DM - I think this is perfect as-is after @spylogsster shared the Chromium comment associated with this value
https://source.chromium.org/chromium/chromium/src/+/main:chrome/common/pref_names.cc;l=1810-1816;drc=44e9423d1b2c17cb40f6953b52b709d48daf2dec

Marking as resolved 😄